api/controllers/api-images-count.js

```diff
@@ -46,22 +46,27 @@ module.exports = {
     }
   },
 
-  fn: async function (inputs,exits) {
+  fn: async function (inputs,exits) {    
 
     if(!this.req.session.userId){
       return exits.forbidden();
     }
 
     // @TODO consolidate this logic into a helper and have this controller share with with the taskQueuer worker
     overrideToZero = false;
-
     let originalFilenameLower = inputs.originalFilenameLower.toLowerCase().trim();
 
-    let query = {
-      originalFilenameLower : {
-        startsWith : originalFilenameLower
-      }
-    };
+    let query = {};
+    if(!originalFilenameLower.includes("*")) {      
+      query.originalFilenameLower = {
+        contains: originalFilenameLower
+      };
+    }  
+    else {      
+      query.originalFilenameLower = {        
+         like: originalFilenameLower.replaceAll("*","%")
+      };
+    }
```

Comment:
tested both and they work as desired - nice!

Reply:
Thank you